external/fv3fit/tests/training/test_train.py

```diff
@@ -27,18 +27,32 @@
     "dense",
     "transformed",
 ]
+
+# unlabeled models that can be tested on all model configuration tests, but not those
+# that actually depend on the quality of supervised learning that occurs
+UNLABELED_TRAINING_TYPES = [
+    "min_max_novelty_detector"
+]
+
 # training functions that have restrictions on the datasets they support,
 # cannot be used in generic tests below
 # you must write a separate file that specializes each of the tests
 # for models in this list
-SPECIAL_TRAINING_TYPES = []
+SPECIAL_TRAINING_TYPES = [
+    "min_max_novelty_detector",
+]
 
 
 # automatically test on every registered training class
-@pytest.fixture(params=GENERAL_TRAINING_TYPES)
+@pytest.fixture(params=GENERAL_TRAINING_TYPES+UNLABELED_TRAINING_TYPES)
 def model_type(request):
     return request.param
 
+# automatically test on every registered training class with labeled data
+@pytest.fixture(params=GENERAL_TRAINING_TYPES)
```

Comment:
To avoid confusion with naming what is now `GENERAL_TRAINING_TYPES` would more accurately be named `SUPERVISED_... or LABELED_...`  and  `GENERAL_... = LABELED.. + NONLABELED_`

Reply:
Regarding this and @nbren12 's comment below, I'm currently going with `REGRESSION_TRAINING_TYPES` and `NON_REGRESSION_TRAINING_TYPES`.